src/components/Input.js

```diff
@@ -1,8 +1,37 @@
 import React from 'react';
-import './Input.css';
+import styled from '@emotion/styled';
+
+const SignInInput = styled.input`
+  width: 250px;
+  border: none;
+  border-radius: 0.5em;
+  padding: 15px 22px;
+  margin: 10px 5px;
+  box-sizing: border-box;
+  background-color: #941e00;
+  color: #f0f0f0;
+  &:focus {
+    background-color: #941e00;
+    color: #f0f0f0;
+  }
+  &::placeholder {
+    color: #f0f0f0;
+  }
+`;
+
+const SignInInputLabel = styled.label`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 85%;
+`;
 
 function Input(props) {
```

Comment:
tl;dr
I would remove the prefix `SignIn` in this file,rename `Input` to `LabeledInput` and add a label text.

long:
This component returns a label and input.
I would rename it to something like `LabeledInput`.

The prefix `SignIn` says that it's connected to a sign-in page or component.
But the file says that it's a generic `Input`.

What is the reason for the label? There is no label or text to display.